helm_deploy/prepare-a-case/templates/_envs.tpl

```diff
@@ -113,4 +113,7 @@ env:
   - name: DEFAULT_TIMEOUT
     value: {{ .Values.env.DEFAULT_TIMEOUT | quote }}
 
+  - name: ENABLE_MATCHER_LOGGING
+    value: {{ .Values.env.ENABLE_APP_INSIGHTS | quote }}
```

Comment:
`.Values.env.ENABLE_APP_INSIGHTS` should be `.Values.env.ENABLE_MATCHER_LOGGING`

Reply:
Great catch! Thanks